engine/ingame/controller.py

```diff
@@ -19,12 +21,19 @@ class InGameController:
     _world: world.World
     _view: view.View
     _keys: Dict[int, bool]
+    api: scripts.GameAPI
+    menu_gui: scripts.GUI
```

Comment:
A pattern I want to start moving towards is using private variables. For any new variables on a class, let's make them private by default unless we have a reason not to.

What that means: prefix the names with an underscore. That's Python's convention for private variables (and is enforced by pylint).